index.html

```diff
@@ -1541,6 +1541,17 @@ <h5>
                     <a data-cite="html/dnd.html#concept-dnd-ro">read-only</a>
                     mode. [[HTML]]
                     </li>
+                    <li>If the pasted content is a file, the {{DataTransfer}}
+                    object's <a data-cite="html/dnd.html#the-drag-data-item-kind">
+                    drag data store item list</a> contains one entry with the
+                    <a data-cite="html/dnd.html#the-drag-data-item-type-string">drag data
+                    item type string</a> being the file's
+                    <a data-cite="mimesniff#mime-type">mime-type</a>, whose
```

Comment:
ReSpec now has a better way to xref: https://respec.org/xref/?term=mime+type

```suggestion
                    item type string</a> being the file's [=mime type=], whose
```

This only works for exported definitions, and it seems HTML does not export the drag data related definition. It's not recommended to use non-exported items since they are assumed as HTML-internal-use-only. I recommend filing an issue/PR.

Reply:
@saschanaz gotcha. Can we create a new issue for this? I assume this will be something we have to do on the entire file and all the references?

Reply:
Sure. I guess having the new syntax just for mime type is enough for this PR, since the DND thing has already been there.